documentation/src/docs/user-guide.template.md

```diff
@@ -1295,6 +1295,55 @@ void restrictedEmailAddresses(@ForAll @Email(quotedLocalPart = false, ipv4Host =
 }
 ```
 
+### Date Arbitrary Types
```

Comment:
Looking at the other headlines I'd rather write "Generation of Dates".

Using annotations (as soons as there are any) will also be documented here so a more generic headline fits better.